### ISSUE TITLE
Editing a patient's parents

### ISSUE THREAD

**Issue by apoorvumang** [apoorvumang/mummy-software#7]

**Comment by apoorvumang** [comment]
The aim is to sync parent data across all siblings.

There are 3 use cases where this needs to be implemented:

1. Adding a new patient and user selects a sibling for them.
2. Patient's profile is opened and her parent's details are changed.
3. Patient's profile is opened and a sibling is added.

**Comment by anuragmahima** [comment]
If parents profile is changed in any one of siblings will it get autocorrected in all siblings, it doesnt happen

**Comment by apoorvumang** [comment]
Yes, it doesn't happen. That's why it is listed as an issue. I'll talk to you later about how you can track issues here.

**Comment by prerakmall** [comment]
Hello aunty! How are you? :dart:

**Comment by apoorvumang** [comment]
Lol arpit did this for her, let's see if she can reply on her own
